src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((prev, singleStyle) => {
+    if (!singleStyle.trim()) {
+      return prev;
+    }
+
+    const [property, value] = singleStyle.split(':');
+
+    if (property && value) {
```

Comment:
The current check `if (property && value)` assumes that both `property` and `value` are non-empty strings. However, if `singleStyle` is something like `':value'`, `property` will be an empty string, and `value` will be 'value'. Consider using `property.trim()` and `value.trim()` in the condition to ensure they are not just whitespace.